src/js/pages/jobs/JobsTable.js

```diff
@@ -42,17 +42,69 @@ class JobsTable extends React.Component {
         prop: 'name',
         render: this.renderHeadline,
         sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
+        sortFunction: this.sortJobNames,
         heading
+      }, {
+        className,
+        headerClassName: className,
+        prop: 'status',
+        render: this.renderStatusColumn,
+        sortable: false,
+        heading
+      }, {
+        className,
+        headerClassName: className,
+        prop: 'lastRunStatus',
+        render: this.renderLastRunStatusColumn
       }
     ];
   }
 
+  getData() {
```

Comment:
I don't see any real performance benefit in this approach over simply calling the getters. But I'd say let's stick with it for the moment, as time is running  and revisit i in the next development cycle. What do you think?